src/components/projects/wbs-details/project-container/project-details/project-details.tsx

```diff
@@ -14,73 +14,61 @@ import {
 } from '../../../../../shared/pipes';
 import PageBlock from '../../../../shared/page-block/page-block';
 import styles from './project-details.module.css';
+import { Col, Container, Row } from 'react-bootstrap';
 
 interface ProjectDetailsProps {
   project: Project;
 }
 
 const ProjectDetails: React.FC<ProjectDetailsProps> = ({ project }: ProjectDetailsProps) => {
   const detailsBody = (
-    <>
-      <div className={styles.halfDiv}>
-        <p>
-          <b>Project Name:</b> {project.name}
-        </p>
-        <p>
-          <b>WBS #:</b> {wbsPipe(project.wbsNum)}
-        </p>
-        <p>
-          <b>Project Lead:</b> {fullNamePipe(project.projectLead)}
-        </p>
-        <p>
-          <b>Project Manager:</b> {fullNamePipe(project.projectManager)}
-        </p>
-        <p>
-          <b>Budget:</b> {dollarsPipe(project.budget)}
-        </p>
-        <div className={styles.horizontal}>
-          <li>{linkPipe('Slide Deck', project.slideDeckLink)}</li>
-          <li>{linkPipe('Task List', project.taskListLink)}</li>
-          <li>{linkPipe('BOM', project.bomLink)}</li>
-          <li>{linkPipe('Google Drive', project.gDriveLink)}</li>
-        </div>
-      </div>
-      <div className={styles.halfDiv}>
-        <p>
-          <b>Duration:</b>{' '}
-          {weeksPipe(project.workPackages.reduce((tot, cur) => tot + cur.duration, 0))}
-        </p>
-        <p>
-          <b>Start Date:</b>{' '}
-          {project.workPackages.length > 0
-            ? project.workPackages
+    <Container fluid className={styles.projectDetails}>
+      <Row>
+        <Col xs={12} md={6}>
+          <div>
+            <b>Project Name:</b> {project.name} <br />
+            <b>WBS #:</b> {wbsPipe(project.wbsNum)} <br />
+            <b>Project Lead:</b> {fullNamePipe(project.projectLead)} <br />
+            <b>Project Manager:</b> {fullNamePipe(project.projectManager)} <br />
+            <b>Budget:</b> {dollarsPipe(project.budget)}
+          </div>
+        </Col>
+        <Col xs={6} md={4}>
+          <div>
+            <b>Duration:</b>{' '}{weeksPipe(project.workPackages.reduce((tot: any, cur: any) => tot + cur.duration, 0))} <br />
+            <b>Start Date:</b>{' '}
+            {project.workPackages.length > 0
+              ? project.workPackages
                 .reduce(
-                  (min, cur) => (cur.startDate < min ? cur.startDate : min),
+                  (min: any, cur: any) => (cur.startDate < min ? cur.startDate : min),
```

Comment:
I think `any` isn't the best type choice here. Is there an issue with the type inference that would require typings? If so, we would want `Date` and something else for the other one